app/pages/help/help-view-model.js

```diff
@@ -0,0 +1,27 @@
+var observableModule = require("data/observable");
```

Comment:
Change all var to const when possible.
only if the variable is going to be reassigned, you can use let